components/Banner.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import styles from "../styles/Banner.module.css";
+
+const Banner = () => {
+  return (
+    <div>
+      <div className={styles.module}>
+        <h2 className={styles.stripe}></h2>
+        <h1 className={styles.text}>UNDER CONSTRUCTION</h1>
```

Comment:
I've pushed a commit, used appropriate tags and instead of new stylsheet I put that some styles in global css
because tailwind css don't have any good alternative for "repeating-linear-gradient()"  for this I have to use custom css. Thanks for comments